### PR TITLE
fixes #530; gateway whitelists not changing if you click the top Save…

### LINKED ISSUE #530: Gateway Settings page does not clear whitelists
Reproduce:

- add origins to whitelist origins array
- confirm it is added in DB
- try and remove whitelist array by submitting a blank field
- items are not cleared from DB

### PR DESCRIPTION
**Pull request by nymd**
The two whitelist fields on the Gateway Settings page had their own "Add" button which really functioned as a "Save" button, but only for those individual fields.

I removed the two "Add" buttons and made the top "Save" button correctly recognize that the whitelist fields have been updated.

### CODE REVIEW

**Review by wgarcia4190 — DISMISSED** [review]

File: frontend/src/views/Apps/GeneralSettings/GeneralSettings.js [hunk 1]
```diff
@@ -42,37 +40,15 @@ class GeneralSettings extends Component {
 
   async saveChanges() {
```
Comment (pabelnl) [inline frontend/src/views/Apps/GeneralSettings/GeneralSettings.js:41]: can we encapsulate everything inside the save changes in a try/catch block?

File: frontend/yarn.lock [hunk 2]
```diff
@@ -1490,10 +1490,10 @@
     "@tendermint/belt" "0.2.1"
     "@tendermint/types" "0.1.1"
 
-"@pokt-network/pocket-js@0.5.8-rc":
-  version "0.5.8-rc"
-  resolved "https://registry.yarnpkg.com/@pokt-network/pocket-js/-/pocket-js-0.5.8-rc.tgz#2df44d0239b479ab3d437f3b74e6d1fe6169ea2f"
-  integrity sha512-L6Aab3cYfRY/DOWAb4W1HJZtY9T/cUgKuVh7cVnwhuL+DNoMVIdl+onwrBGlyGdM91HMU8QO+MfMkTbCK+9uGA==
+"@pokt-network/pocket-js@0.6.1-rc":
+  version "0.6.1-rc"
```
Comment (pabelnl) [inline frontend/yarn.lock]: bump to 0.6.2-rc

**Review by pabelnl — CHANGES_REQUESTED** [review]

File: frontend/src/views/Apps/GeneralSettings/GeneralSettings.js (hunk 1 above)
Comment (nymd) [reply frontend/src/views/Apps/GeneralSettings/GeneralSettings.js:41]: Currently the page only has a "showAlert" function which tells you that the Application has been saved.

Adding the try/catch is trivial but if it does catch an error, it should show some meaningful message to the user.  To do that, we'd need to add another "showError" function to pop up an error.

I think that is out of scope for this bug fix which is being pushed directly to master. But I could be wrong.

Comment (pabelnl) [reply frontend/src/views/Apps/GeneralSettings/GeneralSettings.js:41]: Agree, please leave a Todo comment

File: frontend/yarn.lock (hunk 2 above)
Comment (pabelnl) [reply frontend/yarn.lock]: 0.6.3-rc

**Review by pabelnl — APPROVED** [review]

**Review by wgarcia4190 — APPROVED** [review]

**Pull request merged**